snap/snapcraft.yaml

```diff
@@ -129,6 +130,12 @@ parts:
       usr/bin/git: bin/git
       lib/python3.8/site-packages: lib/
 
+  craftctl:
+    plugin: dump
+    source: libexec
+    organize:
+      craftctl: libexec/charmcraft/craftctl
```

Comment:
this doesn't hurt, but I believe this was only in snapcraft to avoid snapcraft not being able to bootstrap itself.

Reply:
ah, ic the PATH trick

Reply:
Otherwise it executes with the wrong interpreter and doesn't find the necessary packages.